### PR TITLE
do not do picmip on UI elements, fix UnvanquishedAssets/unvanquished_src.dpkdir#21

### PR DESCRIPTION
**Pull request by illwieckz**
Do not do picmip on UI elements, fix UnvanquishedAssets/unvanquished_src.dpkdir#21

Before:

[ui nopicmip](https://dl.illwieckz.net/b/unvanquished/bugs/ui-nopicmip/unvanquished_2020-10-27_173654_000.jpg)

After:

[ui nopicmip](https://dl.illwieckz.net/b/unvanquished/bugs/ui-nopicmip/unvanquished_2020-10-27_173750_000.jpg)

It looks like some UI images used in libRocket things (like the circle menus) are not loaded correctly if I make a `.shader` file, so I did it on game code side.

### CODE REVIEW

**Comment by illwieckz** [comment]
Can we merge this?

File: src/cgame/cg_buildable.cpp [hunk 1]
```diff
@@ -731,8 +731,8 @@ void CG_InitBuildables()
 		//icon
 		if ( ( buildableIcon = BG_Buildable( buildable )->icon ) )
 		{
-		        cg_buildables[ buildable ].buildableIcon = trap_R_RegisterShader( buildableIcon, RSF_DEFAULT );
-                }
+			cg_buildables[ buildable ].buildableIcon = trap_R_RegisterShader( buildableIcon, (RegisterShaderFlags_t) ( RSF_DEFAULT | RSF_NOMIP ) );
```
Comment (slipher) [inline src/cgame/cg_buildable.cpp:734]: `RSF_DEFAULT` is 0, so you don't need the OR or the cast.

**Review by slipher — COMMENTED** [review]
LGTM, it's fine to merge after fixing the comment.

File: src/cgame/cg_buildable.cpp (hunk 1 above)
Comment (illwieckz) [reply src/cgame/cg_buildable.cpp]: Fixed

**Pull request merged**